README.md

```diff
@@ -1,24 +1,123 @@
-# README
+<!-- PROJECT SHIELDS -->
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+[![Contributors][contributors-shield]][contributors-url]
+[![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
 
-Things you may want to cover:
 
-* Ruby version
+<!-- PROJECT LOGO -->
+<br />
+<p align="center">
+  <a href="EPEOLOTRYREPOLINKHERE">
+    <img src="https://user-images.githubusercontent.com/72179421/127752084-8fdb65d3-222d-4daa-a608-4a612440156a.jpg">
+  </a>
 
-* System dependencies
+  <h3 align="center">e . p . e . o . l . a . t . r . y</h3>
 
-* Configuration
+  <p align="center">
+    an app to catalog your literary and vernacular journey. <br> propelling your pronunciation prowess of perplexing prose <br> <em>epi...olive...uh...tree?</em> --> <strong>epeolatry</strong> <br>
+    <br />
+    <a href="https://epeolatrys-rex.herokuapp.com/">View Demo</a>
+    ·
+    <a href="BUGSREPORTLINKHERE">Report Bug</a>
```

Comment:
Is there a way to link to creating a github issue? I feel like that would maybe be the best place to direct both bug report + feature requests

Reply:
I wonder if this will work for non collaborators? 

https://github.com/Caleb1991/epeolatry_front_end/issues/new